deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -269,8 +269,12 @@ cephClusterSpec:
         cpu: "1000m"
         memory: "4Gi"
     prepareosd:
-      # limits: It is not recommended to set limits on the OSD prepare job since it's a one-time burst for memory
-      # that must be allowed to complete without an OOM kill
+      # limits: It is not recommended to set limits on the OSD prepare job
+      #         since it's a one-time burst for memory that must be allowed to
+      #         complete without an OOM kill.  Note however that if a k8s
+      #         limitRange guardrail is defined external to Rook, the lack of
+      #         a limit here may result in a sync failure, in which case a
+      #         limit should be added:  512Mi? 1Gi? Experimentation needed.
```

Comment:
In #11103 we set the limits to 1.2Gi and it still wasn't enough for large disks. So perhaps we recommend to set the limit even to 2Gi in this case.

Reply:
ack.